test/CMakeLists.txt

```diff
@@ -895,6 +897,18 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_sondetac
 # MISC converters
 #===============================================================================
 
+ecbuild_add_test( TARGET  test_iodaconv_tropics
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND bash
+                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                          netcdf
+                  "${CMAKE_BINARY_DIR}/bin/tropics_2ioda.py
+                  -i testinput/tropics_20220216.nc
+                  -o testrun/tropics_20220216.nc4
+                  -d 2022021600"
+                  tropics_20220216.nc4 ${IODA_CONV_COMP_TOL})
+                  
```

Comment:
Seems you used ".nc4" here. Should they be consistent with the testinput and output?

Reply:
I was using the original script from Ben I believe. It took in an nc file and output an nc4. Do you mean lines 907-908? Or do you mean 908 and 910?

Reply:
I thought testoutput should be .nc,  which @PatNichols pointed out and you agreed?

Reply:
this all looks correct now,,, it's a bit confusing that the name of the input and the testoutput and testrun are all so similar, but if the testouput and testrun have some other convention maybe using `20220216T00Z_PT6H` or something make those two files look different

Reply:
@smaticka @BenjaminRuston @huishao-r  The error I saw was that name of the file in the CMakefile did not match the one that was added in this PR. Hence the ctest could not find anything  to compare against.